scripts/bootstrap.sh

```diff
@@ -163,25 +163,6 @@ parse_args() {
   done
 }
 
-sync_repo() {
```

Comment:
reviewers, removing duplicate code. I've consolidated this implementation with the sync_repo in utilities.sh